utility/src/main/java/org/oppia/android/util/networking/NetworkConnectionUtilDebugModule.kt

```diff
@@ -2,12 +2,11 @@ package org.oppia.android.util.networking
 
 import dagger.Binds
 import dagger.Module
-import javax.inject.Singleton
 
 /** Provides debug implementation of [NetworkConnectionUtil]. */
 @Module
 interface NetworkConnectionUtilDebugModule {
-  @Singleton
+  /** Binds [NetworkConnectionUtilDebugImpl] to [NetworkConnectionUtil]. */
```

Comment:
Aha. @Sparsh1212 this might have been a missed case for the KDoc checker. ``@Binds`` should be exempted for the same was ``@Provides`` is.

Reply:
This will be covered in #3648.
Thanks.